src/test/java/emissary/core/TransformHistoryTest.java

```diff
@@ -151,7 +151,7 @@ void testToString() {
 
         String newline = System.getProperty("line.separator");
         StringBuilder expected = new StringBuilder();
-        expected.append("transform history (6) :").append(newline);
+        expected.append("transform history (5) :").append(newline);
```

Comment:
This side-effect is interesting. Do we prefer a total count to include coordinated places?

Reply:
I prefer that coordinated places be included.